build/setup-alpine-tools.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/ash
+#!/bin/bash
```

Comment:
☕️💡question please - why ash (default shell) is dropped and bash is used? Ash is minimalist she’ll also has some good security by design advantage. 

* https://wiki.alpinelinux.org/wiki/Change_default_shell

Reply:
For some reason build systems failed due to this. That's why changed them to bash. This is used by devhost agent, restoration agent and routing manager and uploaded as images to ACR.

Reply:
Thanks bro this is taken care.